src/pages/guidelines/pictograms/usage.mdx

```diff
@@ -55,26 +55,26 @@ It’s important to use pictograms sparingly but effectively throughout your des
 
 ### Sizing
 
-Pictograms are used in a range of sizes, the minimum being 32px while the
+Pictograms are used in a range of sizes, the minimum being 48px while the
```

Comment:
Would the 32px be correct for web usage? It seemed like for the assets we had at 48px they included the padding which ended up with the assets having extra space. It seemed like the 32px sizes were going to be PR'd as a result?

Let me know if this isn't relevant here, too, definitely get that these are used in different applications

Reply:
they're drawn and submitted to the repo at 32px but we recommend using at 48px and up.

Reply:
@conradennis would it help if they were submitted at 48px then but without the padding included? Just wanted to make sure the sizes align up 👀  right now I think the default size in code 32x32 which could cause some confusion / unintended experiences if we want them at 48x48 instead

Reply:
@joshblack might want to check with Mike on that. We updated the guidance on IDL a while back so I think Mike is ok with files being 32px by default since it helps that they're drawn on the same canvas size as icons.